packages/atomic/src/components/insight/user-actions/atomic-insight-user-actions-toggle/e2e/atomic-insight-user-actions-toggle.e2e.ts

```diff
@@ -0,0 +1,32 @@
+import {test, expect} from './fixture';
+
+const exampleUserId = 'exampleUserId';
+const exampleTicketCreationDate = encodeURIComponent('2024-08-30');
+
+test.describe('user actions toggle', () => {
+  test.beforeEach(async ({userActionsToggle}) => {
+    await userActionsToggle.load({
+      args: {
+        userId: exampleUserId,
+        ticketCreationDate: exampleTicketCreationDate,
+      },
+    });
+  });
+
+  test('should display the user actions toggle button', async ({
+    userActionsToggle,
+  }) => {
+    await expect(userActionsToggle.atomicInsightHistoryToggle).toBeVisible();
+  });
+
+  test.describe('when clicking the user actions toggle', () => {
```

Comment:
we should probably add a test or a check to make sure the header is properly displayed as well on top of the timeline.

Reply:
the Atomic Modal is responsible of displaying the header andI believe we have other tests specific to the Atomic Modal.